code/modules/paperwork/contract.dm

```diff
@@ -273,14 +273,16 @@
 	if(!user.dna)
 		return -1
 	user.dna.add_mutation(HULK)
+	user.dna.add_mutation(TK)
```

Comment:
This isn't a downside

Reply:
its not meant to be, trading your soul for what pretty much amounts to hulk and a single not even full heal is not worth it